pkg/k8s/apis/cilium.io/v2/cegp_types.go

```diff
@@ -116,7 +116,7 @@ type EgressGateway struct {
 	// policy will use the first IPv4 assigned to the interface with the
```

Comment:
This comments also needs to be changed if we now support ipv6 egress IPs. I think in general it might be good to add a sentence or two to the documentation that explains behavior and usage of setting an egress IP. Please take a look at `Documentation/network/egress-gateway/egress-gateway.rst` and document the necessary sections and examples with your changes.